takoyaki/src/main/java/frc/robot/subsystems/Hopper.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class Hopper extends SubsystemBase {
+  WPI_TalonSRX floor = new WPI_TalonSRX(Constants.kHooper.MOTOR_ID);
```

Comment:
nit: declare visibility as `private final`
```suggestion
  private final WPI_TalonSRX floor = new WPI_TalonSRX(Constants.kHooper.MOTOR_ID);
```